src/incubator/f5-bigip-ctlr/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
-description: Deploy the F5 Networks k8s-bigip-ctlr 
+description: Deploy the F5 Networks k8s-bigip-ctlr for using the BIG-IP in a Kubernetes or OpenShift environment
```

Comment:
Deploy the F5 Networks BIG-IP Controller for Kubernetes and OpenShift (k8s-bigip-ctlr)

Reply:
Done.